jekyll/_cci2_ja/ecs-ecr.md

```diff
@@ -118,9 +118,9 @@ workflows:
           tag: "${CIRCLE_SHA1}"
       - aws-ecs/deploy-service-update:
           requires:
-            - aws-ecr/build-and-push-image # このジョブは aws-ecr/build-and-push-image が成功した場合のみ実行されます
+            - aws-ecr/build-and-push-image # only run this job once aws-ecr/build-and-push-image has completed
```

Comment:
note: I think this is intentional to keep comments in English for codes (e.g. yml)? 
but some have Japanese comments https://circleci.com/docs/ja/hello-world

please confirm which language we want to use for comments.

Reply:
Sorry for poor explanation, we decided not to translate code comments anymore.
https://circleci.slack.com/archives/CJF173R4H/p1644887332807559
So when changes are made to the code, code comments that were previously translated to Japanese may be reverted to English, but this is allowed.

Reply:
thank for sharing the background. 
I understood the current rule, and agree to be consistent is the most important! ✅